src/Tribe/Main.php

```diff
@@ -557,6 +557,9 @@ public function bind_implementations(  ) {
 			// Ignored Events
 			tribe_singleton( 'tec.ignored-events', 'Tribe__Events__Ignored_Events', array( 'hook' ) );
 
+			// Capabilities
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a28889650b8304343513c3610969a730</sub>